scripts/globals/combat/magic_hit_rate.lua

```diff
@@ -284,6 +284,29 @@ local function magicAccuracyFromFood(actor, magicAccPreFood)
     return magicAcc
 end
 
+-- Global function to calculate magicc accuracy for addiional effects. (No stat, no magic burst bonuses)
```

Comment:
Just a small typo on 'magicc' and 'addiinal' in comment.

Reply:
Corrected